src/front/glsl/parser.rs

```diff
@@ -602,14 +603,15 @@ pomelo! {
 
     layout_qualifier ::= Layout LeftParen layout_qualifier_id_list(l) RightParen {
         if let Some(&(_, loc)) = l.iter().find(|&q| q.0.as_str() == "location") {
-            StructLayout::Binding(Binding::Location(loc))
+            let interpolation = None; //TODO
```

Comment:
GLSL frontend is dysfunctional since we got the `Statement::Emit` in the API. Hopefully, somebody will catch it up with the progress of the IR, but I don't want to include it in scope of this humongous change.